src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/role/RoleAssignmentService.java

```diff
@@ -48,4 +48,12 @@ private RoleAssignmentResource getRoles(String idamUserId, String authToken) {
         }
     }
 
+    public List<RoleAssignment> getRolesByActorId(String actorId,String authToken) {
+        requireNonNull(actorId, "actorId cannot be null");
+
+        RoleAssignmentResource rar = roleAssignmentServiceApi
+            .getRolesByActorId(actorId, serviceAuthTokenGenerator.generate());
+        return rar.getRoleAssignmentResponse();
+    }
+
```

Comment:
Do we need this or can we re-use getRoles()

Reply:
It appears that we can't use getRoles because AM team is adding work types to get a different endpoint, 
in the page attached to the story it says this:
Solution from AM side : Changes to the organisational role mapping service rules and configuration of the role assignment service, as well as logic to build the user’s list of work types.

Configure role assignment service for new work types attribute(s) and this is applicable only for organisation roles.

New Sample mapping rule for ORM:

Updated response of Get by Actor and Query Endpoint of RAS:

Reply:
endpoint removed